src/zone.c

```diff
@@ -278,6 +278,16 @@ static void *Hunk_HighAllocName(int size, char *name)
 	return (void *)(h + 1);
 }
 
+void Hunk_TempFlush()
```

Comment:
Description required, probably, mention what Hunk_TempAllocMore() may require in some cases.